manager/controllers/app/config/evaluator_input.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	api "fybrik.io/fybrik/manager/apis/app/v1alpha1"
+
+	dataset "fybrik.io/fybrik/manager/controllers/app/dataset"
+	"fybrik.io/fybrik/pkg/multicluster"
+	model "fybrik.io/fybrik/pkg/taxonomy/model/base"
+)
+
+// WorkloadInfo holds workload details such as cluster/region, type, etc.
+type WorkloadInfo struct {
+	// Cluster where the user workload is running
+	Cluster multicluster.Cluster
+}
```

Comment:
@shlomitk1 I assume in the future there will be additional information and that this will be a taxonomy driven structure declared somewhere else.  Is this a correct assumption?

Reply:
Yes.